### ISSUE TITLE
Module not found:

### ISSUE THREAD

**Issue by ColeTownsend** [bokuweb/react-rnd#94]
Hey! So this issue is pretty strange – I was previously using version 4.2.0. Since I upgraded, I'm seeing:

```js
Error: Cannot find module 'react-rnd'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/coletownsend/Documents/gits/design-af/.next/dist/components/resizeableDiv.js:39:17)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3) code: 'MODULE_NOT_FOUND'
```

I've tried:
- deleting and reinstalling node modules
- using node `7.2.0` and `7.9.0`
- installing with `npm` instead of `yarn`

**Comment by bokuweb** [comment]
@ColeTownsend Thanks for your report!! sorry.... I'll do investigation... 🙏

**Comment by ColeTownsend** [comment]
@bokuweb it could be a problem with `NextJS` as well, as I downgraded to `4.2.0` and still have the problem

**Comment by ColeTownsend** [comment]
@bokuweb closing this issue as it appears to be a bug with the next.js webpack loader. Downgraded and all works as expected :D

**Comment by bokuweb** [comment]
@ColeTownsend thanks!!

**Comment by bokuweb** [comment]
@ColeTownsend It may be a bug in react-rnd@5.0.0.I forgot putting `main: lib/index.js` in package.json... 🙏 

https://github.com/bokuweb/react-rnd/blob/master/package.json#L5

I 'd published v5.0.1...

Please try using v5.0.1 if you get a chance. 🙇
